internal/ui/app.go

```
@@ -468,13 +611,206 @@ func (m *Model) handleConfirmKey(msg tea.KeyMsg) tea.Cmd {
+		m.pending = nil
+		m.pendingErrs = nil
+		m.state = stateReady
+		m.activity = append(m.activity, "cancelled sudo authentication — nothing was removed")
+		return nil
+	case "enter":
+		pass := m.sudoInput.Value()
+		if pass == "" {
+			m.sudoError = "Password cannot be empty."
+			return nil
+		}
+		cmd := exec.Command("sudo", "-S", "-p", "", "-v")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)